packages/@ama-sdk/schematics/schematics/typescript/core/index.ts

```diff
@@ -36,119 +39,184 @@ const getPathObjectTemplate = (pathObj: PathObject) => {
     }`;
 };
 
+const getGlobalProperties = (globalPropertyMap: Map<string, any>, globalProperties: string) => {
```

Comment:
so far the only available global properties are `debugModels`, `debugOperations`, `stringifyDate`, and `allowModelExtension` which are all booleans, so they can unset one or multiple properties in the command line like this for example: 
`--global-property stringifyDate=false,allowModelExtension=false`

Reply:
We specify in the documentation that we are helping them because in fact the call the generator with `--generator-key` which means it should rely only on the openapitools.json and raise an error if there is a definition in the json and in the cli.
For me it's a case we don't want to handle 
people can still change the value in their openapitools.json if they want to unset a specific property